packages/svelte-check/src/index.ts

```diff
@@ -30,11 +30,43 @@ async function openAllDocuments(
     filePathsToIgnore: string[],
     svelteCheck: SvelteCheck
 ) {
-    const files = await glob('**/*.svelte', {
-        cwd: workspaceUri.fsPath,
-        ignore: ['node_modules/**'].concat(filePathsToIgnore.map((ignore) => `${ignore}/**`))
+    const offset = workspaceUri.fsPath.length + 1;
+    // We support a very limited subset of glob patterns: You can have a * at the end or the start
+    const ignored: Array<(path: string) => boolean> = filePathsToIgnore.map((i) => {
+        if (i.endsWith('*')) i = i.slice(0, -1);
+        if (i.startsWith('*')) {
+            i = i.slice(1);
+            if (i.includes('*'))
+                throw new Error(
+                    'Invalid svelte-check --ignore pattern: Only * at the start or end is supported'
+                );
+            return (path) => path.includes(i);
```

Comment:
I think this needs to be posixified. If someone checks in something like `"check": "svelte-check --ignore packages/fixtures/**"` and then their Windows teammate runs it I think this could break

Reply:
this is handled by the `withPathSeparator('/')`, which does the posification for us